### PR TITLE
MAYA-103409 Minimal Create USD Stage

### PR DESCRIPTION
**Pull request by luceric-adsk**
A user currently can't create a USD stage using the UI. This UI will allow the user to create a stage using an existing USD layer.

In Maya's top level menu, under Create, a new entry 'Universal Scene Description (USD)' is underneath 'Scene Management' category.  Please have this in the What's New green highlight style.  On hover, the submenu appears with the following entry:
"Stage from Existing Layer..."
Please have this in the What's New green highlight style.   On click, the following file dialogue appears[...]

### CODE REVIEW

File: lib/nodes/proxyShapeBase.cpp [hunk 1]
```diff
@@ -177,6 +178,20 @@ MayaUsdProxyShapeBase::initialize()
     retValue = addAttribute(excludePrimPathsAttr);
     CHECK_MSTATUS_AND_RETURN_IT(retValue);
 
+    loadPayloadsAttr = numericAttrFn.create(
+        "loadPayloads",
+        "lpl",
+        MFnNumericData::kBoolean,
+        0.0,
```
Comment (mattyjams) [inline lib/nodes/proxyShapeBase.cpp]: Shouldn't this attribute be turned on by default? That would preserve existing behavior of loading payloads when opening the stage.

Comment (luceric-adsk) [reply lib/nodes/proxyShapeBase.cpp]: let me fix that by doing in the definition line instead

**Comment by luceric-adsk** [comment]
added support for case when user is using native dialog

File: lib/nodes/proxyShapeBase.cpp (hunk 1 above)
Comment (mattyjams) [reply lib/nodes/proxyShapeBase.cpp]: Oh, sorry! I read right past that `setDefault(true)` call.

But yeah, looks a bit cleaner to just have it in the initial definition. Thanks!

**Pull request merged**